server/src/compression.rs

```diff
@@ -468,8 +515,8 @@ mod tests {
 
     #[test]
     fn test_try_compress_regular_linear_time_series() {
```

Comment:
Tests for irregular linear time series seems to be missing.

Reply:
Fixed.

Reply:
`test_try_compress_irregular_almost_linear_time_series` still seems to be missing.

Reply:
Fixed.